Users/Install.rst

```diff
@@ -57,6 +57,9 @@ system-wide, use:
 
 .. note::
 
+    **Ubuntu Users** - Don't use **apt** command for installing coala
+    It's a different software
```

Comment:
i bump this comment. also, it should be "don't use ``apt-get install coala`` for installing coala as that's a different software." (note the trailing period)